encrypted-media-respec.html

```diff
@@ -1011,11 +1011,30 @@
         <dd>
           <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/85">Issue 85</a> - There are concerns about the architectural implications of <code>"persistent-usage-record"</code> sessions that are pending a TAG discussion. The outcome of this discussion could result in modification (including removal) of the feature.</p>
           <p>
-            A session for which the license and any key(s) it contains SHALL NOT be persisted and for which a <var>record of key usage times</var> SHALL be persisted.
-            The <var>record of key usage times</var> consists of a record of the key IDs of the keys used in the session, the <var>first decryption time</var> for each key and the <var>latest decryption time</var> for each key.
-            A <a def-id="message"></a> of type <a def-id="message-type-license-release"></a> containing a record of key usage will be generated when <a def-id="remove"></a> is called and on every <a def-id="load"></a> call until the record is acknowledged by a response passed to <a def-id="update"></a>. Because the license and keys are not persisted, this record implicitly proves that the keys are no longer available in the session.
+            A session for which the license and any key(s) it contains SHALL NOT be persisted and for which a <a def-id="record-of-key-usage"></a> SHALL be persisted.
+            The <dfn id="record-of-key-usage">record of key usage</dfn> consists of:
+          </p>
+          <ul>
+            <li>
+              <p>A record of the <a def-id="key-id">key IDs</a> of all the key(s) that were at any time <a href="#known-key">known</a> to the session,</p>
+            </li>
+            <li>
+              <p>
+                The <dfn id="first-decryption-time">first decryption time</dfn>, defined as the time at which the session was first use to decrypt content, and
```

Comment:
Yes, that is what I meant here. Yes, I suggested `<dfn>`, but when I looked at other examples, that didn't seem appropriate. We use `<dfn>` for terms like **Initialization Data Type** but not for internal/local variables like this.

I think it would be fine to define "First Decryption Time" if you want to reference the term elsewhere, but I believe the internal variable should be a `<var>`.

You could probably handle both here by saying something like the following. It's not exactly the best structure, but it works.
```<var>first decryption time</var> - The <dfn id="first-decryption-time">first decryption time</dfn>, which is the time at which the session was first use to decrypt content.```